cardano-cli/src/Cardano/CLI/Shelley/Run/Query.hs

```diff
@@ -380,6 +490,132 @@ writeLedgerState mOutFile qState@(SerialisedLedgerState serLedgerState) =
      first (const ls) (decodeFull ls)
 
 
+-- Data Structure to hold stake information so that it can be printed nicely
+data Stakes =  Stakes {
```

Comment:
Let's add some docs as to that `mark, set, go, total` mean. Both here and in the cli docs.
`-- Data Structure to hold stake information so that it can be printed nicely` is unnecessary and should be deleted.

Reply:
I think it's important to explain why this data structure is introduced here.  It really has no purpose other than displaying the stake information.  Generally, I find it's good practice to comment data structures.  I have added some comments about mark, set, go.  They're a bit esoteric